Formula/d/distill-cli.rb

```diff
@@ -0,0 +1,26 @@
+class DistillCli < Formula
+  desc "Use AWS Transcribe and Bedrock to create summaries of your audio recordings"
+  homepage "https://www.allthingsdistributed.com/2024/06/introducing-distill-cli.html"
+  url "https://github.com/awslabs/distill-cli.git",
+      using:    :git,
+      revision: "294c94ba552f4a5e40478567adffca4de0beaf84",
+      tag:      "v0.1.0-beta"
+  license "Apache-2.0"
+  head "https://github.com/awslabs/distill-cli.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # Most of the functionality is in the cloud, so
+    # our testing options are basically limited to
+    # ensuring the binary runs on the local system.
+    #
+    # Need to create a config file or cli will fail
+    (testpath/"config.toml").write ""
+    shell_output "#{bin}/distill-cli --help"
```

Comment:
wonder the artifact name should be better as distill rather than distill-cli (can be escalated onto aws side)

Reply:
Good point, I will bring it up with them.

Reply:
I asked. They won't be changing it for reasons I won't go in to.